app/helpers/pages_helper.rb

```diff
@@ -1,2 +1,5 @@
 module PagesHelper
+  def full_name(first_name, last_name)
+    "#{first_name} #{last_name}"
+  end
```

Comment:
Les helper c'est rapidement fourre-tout. Toutes les méthodes des modules de helpers sont mélangées et accessible partout. Après on ne peut pas faire la distinction entre une méthode `full_name` qui concerne un user ou une méthode `full_name` qui concerne un autre modèle.

Du coup on limite les helpers aux méthodes qui concernent vraiment les vues de manière générique, comme ceux que propose rails de base : `link_to`, `content_tag`, `form_for` etc. La logique qui n'est pas spécifique à un modèle.

Cette méthode `full_name` qui concerne le user pourrait aller dans le modèle `User`.

Si on ne veut pas encombrer le modèle avec des méthodes qui ne servent à présenter les données, on peut faire un décorateur. Draper est excellent pour ça : https://github.com/drapergem/draper

Apparemment la formation ne parle pas de draper, donc autant l'installer tout de suite, ça sera aussi utile plus tard.

Reply:
Ok c'est bien noté, je vais m'essayer au décorateur alors :)

Reply:
Done